guides/v2.3/graphql/queries/directory-country.md

```diff
@@ -0,0 +1,121 @@
+---
+group: graphql
+title: country query
+redirect_from:
+  - /guides/v2.3/graphql/reference/directory.html
+---
+
+Use the `country` query to retrieve information about a specific country.
+
+## Syntax
+
+`{country(id: String) {Country}}`
+
+## Example usage
+
+The following query returns information about Australia.
+
+**Request** 
+
+```text
+query {
+    country(id: "AU") {
+        id
+        two_letter_abbreviation
+        three_letter_abbreviation
+        full_name_locale
+        full_name_english
+        available_regions {
+            id
+            code
+            name
+        }
+    }
+}
+```
+
+**Response**
+
+```json
+{
+  "data": {
+    "country": {
+      "id": "AU",
+      "two_letter_abbreviation": "AU",
+      "three_letter_abbreviation": "AUS",
+      "full_name_locale": "Australia",
+      "full_name_english": "Australia",
+      "available_regions": [
+        {
+          "id": 569,
+          "code": "ACT",
+          "name": "Australian Capital Territory"
+        },
+        {
+          "id": 570,
+          "code": "NSW",
+          "name": "New South Wales"
+        },
+        {
+          "id": 576,
+          "code": "NT",
+          "name": "Northern Territory"
+        },
+        {
+          "id": 572,
+          "code": "QLD",
+          "name": "Queensland"
+        },
+        {
+          "id": 573,
+          "code": "SA",
+          "name": "South Australia"
+        },
+        {
+          "id": 574,
+          "code": "TAS",
+          "name": "Tasmania"
+        },
+        {
+          "id": 571,
+          "code": "VIC",
+          "name": "Victoria"
+        },
+        {
+          "id": 575,
+          "code": "WA",
+          "name": "Western Australia"
+        }
+      ]
+    }
+  }
+}
+```
+
+## Input attributes {#inputAttributes}
+
```

Comment:
The Input attributes section is limited to `id`

The Output attributes section should say "The query returns a single `Country` object. Then add a reference to the include file.

Reply:
I put the description plus headers in the include file.

Reply:
Cleaned this up after talking to Kevin.